src/librustc_error_codes/error_codes/E0725.md

```diff
@@ -4,8 +4,8 @@ command line flags.
 Erroneous code example:
 
 ```ignore (can't specify compiler flags from doctests)
-#![feature(specialization)] // error: the feature `specialization` is not in
-                            // the list of allowed features
+#![feature(never_type)] // error: the feature `never_type` is not in
```

Comment:
Keep `#![feature(specialization)]` to make things easier the second time around.